lib/yi_appium_caps_util.rb

```diff
@@ -6,14 +6,14 @@
 class YiAppiumCapsUtil
   class << self
     public
-    def update (caps_file_name = './appium.txt')
+    def update (caps_file_name: './appium.txt', platformName_value: nil)
```

Comment:
Why did you have to change `=` to `:`?

Reply:
won't I be forced to populate caps_file_name first if I keep it with '='?

Reply:
Then if thats the case you might not need all the ifelses in the `update` call, just call it once with the same parameters. I like the use `:`